esmvaltool/cmorizers/obs/cmorize_obs_glodap.py

```diff
@@ -0,0 +1,133 @@
+"""ESMValTool CMORizer for GLODAP data.
+
+Tier
+   Tier 2: other freely-available dataset.
+
+Source
+   GLODAP: https://www.glodap.info/index.php/mapped-data-product/
+
+Last access
+   GLODAP: 20210528
+
+Download and processing instructions
+   All handled by the script (download only if local raw data are missing)
+
+Modification history
+   20210528-lovato_tomas: written
+
+"""
+
+import logging
+import os
+from warnings import catch_warnings, filterwarnings
+import tarfile
+import requests
+import iris
+from cf_units import Unit
+
+from .utilities import (constant_metadata, fix_coords, fix_var_metadata,
+                        save_variable, set_global_atts)
+
+logger = logging.getLogger(__name__)
+
+
+def _fix_data(cube, var):
+    """Specific data fixes for different variables."""
+    logger.info("Fixing data ...")
+    with constant_metadata(cube):
+        if var in [
+                'dissic',
+                'talk',
+        ]:
+            cube /= 1000.  # Convert from umol/kg to mol/m^3
+    return cube
+
+
+def collect_files(in_dir, var, cfg):
+    """Compose input file list and download if missing."""
+    var_dict = cfg['variables'][var]
+
+    fname = '.'.join([var_dict['file'], var_dict['raw_var'], 'nc'])
+    in_file = os.path.join(in_dir, fname)
+
+    # check if input file is missing
+    if not os.path.isfile(in_file):
```

Comment:
In fact you do not need to bother with downloading the data automatically. This feature will be added with #1657 and we can pick up from there to include a downloading script for GLODAP using the framework that has been developed there. My guess is that this part will be removed when we include this to #1657.

Reply:
I think that it will be necessary to update the cmorizers in any case after #1657 to use it for downloading the data.